React/CoreModules/RCTLogBox.mm

```diff
@@ -72,11 +72,95 @@ - (void)show
 
 @end
 
+#else // [TODO(macOS ISS#2323203)
+
+@interface _RCTLogBoxView : NSView
+@end
+
+@implementation _RCTLogBoxView
+- (BOOL)isFlipped
+{
+  return YES;
+}
+- (NSView *)hitTest:(NSPoint)point
+{
+  return self.subviews[0];
+}
+@end
+
+@interface RCTLogBoxWindow : NSWindow
+@end
+
+@implementation RCTLogBoxWindow {
+  RCTSurface *_surface;
+}
+
+- (instancetype)initWithBridge:(RCTBridge *)bridge
+{
+  NSRect minimumFrame = NSMakeRect(0, 0, 600, 800);
+  NSRect maximumFrame = minimumFrame;
+
+  if ((self = [self initWithContentRect:minimumFrame
+                              styleMask:NSWindowStyleMaskTitled
+                                backing:NSBackingStoreBuffered
+                                  defer:YES])) {
+    _surface = [[RCTSurface alloc] initWithBridge:bridge moduleName:@"LogBox" initialProperties:@{}];
+
+    [_surface start];
+    // TODO: It is not entirely clear to me why there needs to be a range. For now we'll use a fixed size.
+    [_surface setMinimumSize:minimumFrame.size maximumSize:maximumFrame.size];
```

Comment:
The iOS version just calls setSize:.   That doesn't work on mac?

Reply:
I think it did. I’ll check and revert to that 👍 

I switched to this as I was testing out what it was and if resizing the window makes sense—except it didn’t as the size of the RN layout remains fixed, just at the maximum size.